allure-generator/src/main/javascript/blocks/label/styles.scss

```diff
@@ -11,5 +11,7 @@
 
   &__info {
     background: #3c58e4;
+    margin: 1px;
```

Comment:
that is bad place to add margin and change display format, because this block can be used in many different places. Also info label should have the same margin as other labels such as failed or broken.

Reply:
ok thanks, I will investigate the best solution for this, any recommendations here?